test/orderController.test.js

```diff
@@ -0,0 +1,137 @@
+const { app, connectDB } = require('../src/expressServer'); // Import your Express app here
+const { StatusCodes } = require('http-status-codes');
+const request = require('supertest');
+const { MongoMemoryReplSet } = require('mongodb-memory-server');
+const User = require('../src/models/User');
+const Album = require('../src/models/Album');
+const { loginAndReturnCookie } = require('./test_helper');
+
+let server;
+let mongooseConnection;
+let mongodb;
+
+jest.mock('stripe', () => {
+  // When `require('stripe')` is called, the below jest mock function will be returned instead
+  return jest.fn(() => ({
+    // When the `require('stripe')(API_KEY)` is called, the below object will be returned instead
+    paymentIntents: {
+      // The `stripe.paymentIntents.create` will be replaced with the below jest mock function that returns our mock data
+      create: jest.fn(() => ({
+        id: 'pi_mock123',
+        client_secret: 'mock_secret',
+      })),
+    },
+  }));
+});
+
+beforeAll(async () => {
+  mongodb = await MongoMemoryReplSet.create({
+    replSet: { storageEngine: 'wiredTiger' },
+  });
+  const url = mongodb.getUri();
+  process.env.MONGO_URL = url;
+  mongooseConnection = await connectDB(url);
+  server = await app.listen(8001);
+  // process.env.STRIPE_SECRET_KEY = stripeSecretKey; -Akos: I didn't understand what else to do using this approach
```

Comment:
This line can be removed :+1:  Let me know if you have any remaining questions about the approach used here to mock the stripe module :slightly\_smiling\_face: